WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -345,6 +347,29 @@ struct ReaderPostMenuButtonTitles {
         dispatchToggleFollowSiteMessage(siteTitle: topic.title, siteID: topic.siteID, following: topic.following, success: success)
     }
 
+    class func dispatchToggleSubscribeCommentMessage(subscribing: Bool, success: Bool) {
+        let title: String
+        if success {
+            title = subscribing ?
+                NSLocalizedString("Successfully followed conversation", comment: "The app successfully subscribed to the comments for the post") :
```

Comment:
Nitpick: There's a `NoticeMessages` struct that contains the other notice messages. Maybe these can be moved into that?

Reply:
Thank you!